lib/connection/messages.coffee

```diff
@@ -75,3 +76,19 @@ module.exports =
       else
         ""
       dismissable: true
+
+  connectExternal: ->
+    tcp.listen().then (port) ->
+      msg = atom.notifications.addInfo "Connect an external process",
+        detail: """
+        To connect a Julia process running in the terminal,
+        run the command:
+
+            using Juno
+            Juno.connect(#{port})
+        """
+        dismissable: true
```

Comment:
Great idea, I've added this. It's something we can think about applying to our other buttons as well.